src/celengine/star.h

```diff
@@ -229,8 +229,9 @@ StarDetails::getEllipsoidSemiAxes() const
 bool
 StarDetails::hasCorona() const
 {
-    // Y dwarves and T dwarves subclasses 5-9 don't have a corona
-    return spectralType[0] != 'Y' && (spectralType[0] != 'T' || spectralType[1] < '5');
+    // Don't render corona for brown dwarfs that are not hot enough to glow.
+    // See https://en.wikipedia.org/wiki/Draper_point
+    return temperature >= 798;
```

Comment:
i don't think this law can be applied here

Reply:
It's better than an arbitrary cutoff at T5.

Reply:
Sun's corona has temperature of millions of K

Reply:
> The absence of X-rays from LP 944-20 during the non flaring period is also a significant result. It sets the lowest observational limit on steady X-ray power produced by a brown dwarf star, and shows that million degree Celsius upper atmospheres, or coronas, cease to exist as the surface temperature of a brown dwarf cools below about 2500 degrees Celsius and becomes electrically neutral.

https://chandra.harvard.edu/xray_sources/browndwarf_fg.html

Reply:
@SevenSpheres do you have any updates for this pr?

Reply:
Removed that commit, will open a separate issue later